### ISSUE TITLE
Suggestion: Multiple columns of tags in Taxonomies Tab

### ISSUE THREAD

**Issue by pixelstuff** [craftsmancoding/taxonomies#27]
Let's say I have a list of 50 tags. That gets really long on the Taxonomies tab with a lot of space wasted to the right.

Can we we have an option to tell the Taxonomy group to display on the Taxonomies tab using either multiple columns (wrapping) or in a single column (as it does now)? Make it per taxonomy group if possible. If not, then a global setting might be workable.

**Comment by fireproofsocks** [comment]
Good idea.  The only reason the current formatting was used was simplicity (that and it's easier to spot terms if they're sorted, e.g. alphabetically).  Thoughts on a tag cloug?  Or Ajax style type-ahead search to filter?
